addons/portal/views/portal_templates.xml

```diff
@@ -23,7 +23,7 @@
                             <t t-call="portal.user_dropdown">
                                 <t t-set="_user_name" t-value="true"/>
                                 <t t-set="_item_class" t-value="'nav-item dropdown'"/>
-                                <t t-set="_link_class" t-value="'nav-link'"/>
+                                <t t-set="_link_class" t-value="'btn nav-link'"/>
```

Comment:
@Brieuc-brd My point was to **not** use the `btn` class alongside the `nav-link` class :thinking: I don't understand your changes :thinking:

Reply:
Sorry @qsm-odoo, since the purpose of this PR is to improve the layout of `header_sales_three` especially, I thought the goal was to only remove the `btn` class for this template and keep the current behavior for the others.
I think I can create another commit to fix the other iterations so?

Reply:
Oh you are in fact keeping the same behavior by doing this? That's ok I guess...:) But indeed, it would be better to directly add an extra commit to fix that if possible :pray: